metpy/calc/thermo.py

```diff
@@ -2155,3 +2155,63 @@ def dewpoint_from_specific_humidity(specific_humidity, temperature, pressure):
     return dewpoint_rh(temperature, relative_humidity_from_specific_humidity(specific_humidity,
                                                                              temperature,
                                                                              pressure))
+
+
+@exporter.export
+@preprocess_xarray
+@check_units('[length]/[time]', '[mass]/[volume]')
+def vertical_velocity_height_to_pressure_hydrostatic(w, density):
```

Comment:
Would it make sense to default to our constants `rho_d` in the signature here?

Reply:
I would lean against it, since this is likely a calculation done throughout some depth of the atmosphere, and the changes in density with height are significant, so having it default to a constant may be misleading/inaccurate.